snowexsql/tables/instrument.py

```diff
@@ -10,3 +11,15 @@ class Instrument(Base):
     name = Column(String(), index=True)
```

Comment:
I missed this in the last PR, but ID on line 8 can be removed

Reply:
Yep good call